cli/module_registry.go

```diff
@@ -114,34 +114,31 @@ func UpdateModuleCommand(c *cli.Context) error {
 	orgIDArg := c.String("org_id")
 	manifestPathArg := c.String("module")
 
-	client, err := NewAppClient(c)
-	if err != nil {
-		return err
-	}
 	manifestPath := defaultManifestFilename
 	if manifestPathArg != "" {
 		manifestPath = manifestPathArg
 	}
-	if _, err := os.Stat(manifestPath); err != nil {
-		return errors.Wrapf(err, "Cannot find %s", manifestPath)
-	}
 
-	manifestBytes, err := os.ReadFile(manifestPath)
+	client, err := NewAppClient(c)
 	if err != nil {
 		return err
 	}
-	var manifest ModuleManifest
-	if err := json.Unmarshal(manifestBytes, &manifest); err != nil {
+
+	manifest, err := loadManifest(manifestPath)
+	if err != nil {
 		return err
 	}
+
 	moduleID, err := parseModuleID(manifest.Name)
 	if err != nil {
 		return err
 	}
+	// TODO(zaporter) This logic is duplicated in update and upload and should be refactored
```

Comment:
nit: delete? or make a ticket?

Reply:
This will be done as part of the simplifications that come from `APP-2230`. Changed message to reflect that.